src/tests/ftest/launch.py

```diff
@@ -953,6 +958,161 @@ def rename_logs(avocado_logs_dir, test_file):
                 test_logs_dir, new_test_logs_dir, error))
 
 
+USE_DEBUGINFO_INSTALL = True
+
+def install_debuginfos():
+    """Install debuginfo packages"""
+
+    install_pkgs = [{'name': 'gdb'}, {'name': 'python-magic'}]
+    cmds = []
+
+    if USE_DEBUGINFO_INSTALL:
+        cmds.append("sudo debuginfo-install -y "                   \
+                    "--exclude ompi-debuginfo,gcc-debuginfo,"      \
+                               "gcc-base-debuginfo "               \
+                    "daos-server cart libpmemobj python openmpi3")
+    else:
+        import yum
```

Comment:
PEP8 recommends placing all imports at the top of the file: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Yes, I know this.  I put this here so that the yum module is not necessary if the user doesn't want to do core processing, keeping the bar to being able to run `launch.py` only has high as the options the user wants.

Reply:
Without the import at the beginning of the file the user may not no it's missing until it's too late - and in this case it will kill launch.py with an ImportError exception - possibly in the middle of testing.

Reply:
Yeah.  That is the downside.  IMHO, it's better than raising the price-to-play for those that don't use the option.

I suppose the `import` could be moved to be activated (and error raised if missing) when the `-p` argument is detected, before anything else happens, but that seems like it would be icky.